Assets/Mirror/Transports/KCP/kcp2k/highlevel/KcpPeer.cs

```diff
@@ -661,9 +729,18 @@ void SendUnreliable(ArraySegment<byte> message)
         // => handshake info needs to be delivered, so it goes over reliable.
         public void SendHandshake()
         {
+            // server includes a random cookie in handshake.
+            // client is expected to include in every message.
+            // this avoid UDP spoofing.
+            // KcpPeer simply always sends a cookie.
+            // in case client -> server cookies are ever implemented, etc.
```

Comment:
I don't see the point of server sending the cookie in all outbound...that's just 4 more bytes we don't need.  I can't imagine a scenario where we'd add this.